packages/sdk/src/sdk/api/challenges/types.ts

```diff
@@ -53,35 +53,28 @@ const DefaultSpecifier = z.object({
 const ReferralSpecifier = z.object({
   /** The challenge identifier. As in, the challenge "name." */
   challengeId: z.enum([ChallengeId.REFERRALS, ChallengeId.VERIFIED_REFERRALS]),
-  /** The user ID of the user that was referred. */
-  referredUserId: HashId,
   /** The user ID of the user completing the challenge. */
-  userId: HashId
-})
-
-const TrackSellerSpecifier = z.object({
-  /** The challenge identifier. As in, the challenge "name." */
-  challengeId: z.enum([ChallengeId.AUDIO_MATCHING_BUYER]),
-  /** The user ID of the owner of the track purchased. */
-  sellerUserId: HashId,
-  /** The track ID that was purchased. */
-  trackId: HashId
+  userId: HashId,
+  /** The user ID of the user that was referred. */
+  referredUserId: HashId
 })
 
-const TrackBuyerSpecifier = z.object({
+const AudioMatchSpecifier = z.object({
```

Comment:
they are technically separate rewards- not sure why we should combine, but your call

Reply:
Same with rewards - this schema looks the same for all audio matching reward specifier args so it simplifies the generated types a bit

Reply:
in other words, this just the schema for the generateSpecifier function args when it's an audio match reward, not actually defining the type of the reward specifier itself